hyperdome_client/settings_dialog.py

```diff
@@ -57,9 +58,9 @@ def __init__(self, common: Common,
         self.common.log('SettingsDialog', '__init__')
 
         self.onion = onion
-        self.qtapp = qtapp
-        self.config = config
-        self.local_only = local_only
+        self.qtapp: QtWidgets.QApplication = qtapp
+        self.config: bool = config
```

Comment:
ugh, the reason I added that type hint is because I drove myself nuts trying to figure out what config meant, it should really just be "has_config".

Fixing